static/js/src/entry/account/routes/membership/components/RecurringOrCircle.vue

```diff
@@ -1,13 +1,37 @@
 <template>
   <section class="c-detail-box">
     <div class="has-xxl-btm-marg">
+      <p
+        v-if="declinedCard"
+        role="alert"
+        class="has-b-btm-marg has-text-error"
+      >
+        <strong>Card was declined</strong>
+      </p>
```

Comment:
I need to think through portal-api being down or hiccuping... think that would be applicable to the profile update form as well, as you mentioned, so that can PROBABLY be considered outside of this project BUT I did make some updates to error handling in the case where portal-api doesn't get the exact pieces it's currently expecting to get, which should clear up some of that "Wait, why is my card reading as declined?"